src/Options.ts

```diff
@@ -157,7 +157,7 @@ function getRequirableModulePath(modulePath: string): string {
     return resolve(modulePath);
   }
 
-  for (let ext of SUPPORTED_EXTENSIONS) {
+  for (let ext of RequireableExtensions) {
```

Comment:
`ext` can be `const`

Reply:
In general I've mostly followed Ember's conventions for `let` and `const`, but lately I've been tempted to just use `const` everywhere since it is sometimes needed for TypeScript to make certain guarantees when type checking. For now I'll leave these as `let` and maybe change them all in a later refactor.